src/app/cli/src/cli_parser.rs

```diff
@@ -131,7 +131,7 @@ pub fn cli() -> Command {
                             .help("Dataset manifest reference(s) (path, or URL)"),
                     ])
                     .after_help(indoc::indoc!(
-                        r#"
+                        r"
```

Comment:
These fixes seem strange to me. Yes the `r#` may not be necessary now, but the second we add a double quote to this doc string we'll have to change it back to `r#"..."#`. I see it as future-proofing documentation against larger diffs than necessary.

Reply:
Will disable this rule